code/game/objects/effects/spawners/minor_variation_templates.dm

```diff
@@ -0,0 +1,75 @@
+// sec
```

Comment:
```suggestion

```
Change it to something better, "sec" just... just bad

Reply:
It's literally just a 3 letter comment. 
What do you want this to be something like // ===== SECURITY  =====